lib/Reader/readerir.cpp

```diff
@@ -1879,11 +1902,13 @@ bool GenIR::isUnmanagedPointerType(llvm::Type *Type) {
 //
 //===----------------------------------------------------------------------===//
 
-EHRegion *fgNodeGetRegion(FlowGraphNode *Node) { return nullptr; }
-
-EHRegion *fgNodeGetRegion(llvm::Function *Function) { return nullptr; }
+EHRegion *GenIR::fgNodeGetRegion(FlowGraphNode *Node) {
+  return FlowGraphInfoMap[Node].Region;
+}
 
-void fgNodeSetRegion(FlowGraphNode *Node, EHRegion *Region) { return; }
+void GenIR::fgNodeSetRegion(FlowGraphNode *Node, EHRegion *Region) {
+  FlowGraphInfoMap[Node].Region = Region;
```

Comment:
Sure.  I added the assert, but I already had to change [this caller](https://github.com/JosephTremoulet/llilc/commit/033ce255f4bd8dd129441d6514f1cd103b44e8b8#diff-f3d514eb35910c5b4b5e3c8b5af72317R2987) to use the Update variant -- we create these blocks by successively splitting one big block, and splitBlock annotates each newly-created tail with the splittee's region, so when we first get to each region we're actually overriding the previous region on that block.  (you can see just what I changed [here](https://github.com/dotnet/llilc/compare/62fd29482...JosephTremoulet:b86c9df)).